--- webapps/omnigator/views/template_no_frames.jsp
+++ webapps/omnigator/views/template_no_frames.jsp
@@ -45,7 +45,7 @@
           <td align="right" valign="top" width="300">
             <!-- Right -->
             <div id="oksmenu">
-              <a href="/" title="Go to application home page.">Home</a>&nbsp;|&nbsp;<a href="/manage/manage.jsp" title="Refresh sources, reload settings, manage plug-ins, create indexes, etc.">Manage</a>&nbsp;|&nbsp;<a href="http://www.ontopia.net/" title="Go to the Ontopia website.">Website</a>&nbsp;|&nbsp;<a href="http://groups.google.com/group/ontopia" title="Mailing list.">Support</a>&nbsp;|&nbsp;<a href="/about.jsp">About</a>
+              <a href="http://www.ontopia.net/" title="Go to the Ontopia website.">Website</a>&nbsp;|&nbsp;<a href="http://groups.google.com/group/ontopia" title="Mailing list.">Mailing list</a>&nbsp;|&nbsp;<a href="https://github.com/ontopia/ontopia/issues" title="Support.">Support</a>
             <%
               String tm = request.getParameter("tm");
               if (tm != null && tm.equals("null"))
